cypress/e2e/models/administration/credentials/credentials.ts

```diff
@@ -140,20 +141,30 @@ export class Credentials {
     }
 
     protected verifyDefaultCredentialIcon(): void {
-        if (this.isDefault) {
-            cy.get(tdTag, { timeout: 120 * SEC })
-                .contains(this.name, { timeout: 120 * SEC })
-                .closest(trTag)
-                .within(() => {
+        const isDefault = this.isDefault; // Capture the value before the Cypress command chain
+        cy.log("this.isDefault", isDefault);
+        cy.get(tdTag, { timeout: 120 * SEC })
+            .contains(this.name, { timeout: 120 * SEC })
+            .closest(trTag)
+            .within(() => {
+                if (isDefault) {
+                    cy.log("this.isDefault exist", isDefault);
                     cy.get("td").eq(1).find("svg.pf-v5-svg").should("exist");
```

Comment:
"svg.pf-v5-svg" could be moved to credentials.view.ts since it's being used in other places as well.